lib/screens/episode.dart

```diff
@@ -356,6 +360,10 @@ class _EpisodeDisplay extends HookConsumerWidget {
 
     final showLoadingOverlay = (episodeSnapshot.connectionState == ConnectionState.waiting);
 
+    if (enableAutoplayNext && episodeIsCurrentItem && !playerSetupSnapshot.hasError && playerSetupSnapshot.connectionState == ConnectionState.done) {
+      playbackService.getNextEpisodeForPlayer(playerId: player.playerId).then((value) => nextEpisode.value = value);
```

Comment:
Be mindful what you put directly in build() :)
Wrap in memoized to prevent querying the backend every build().
Also check if context.mounted before assigning values.